bundle-workflow/src/assemble_workflow/bundle_recorder.py

```diff
@@ -2,23 +2,32 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import os
+from urllib.parse import urljoin
+
 import yaml
+
 from manifests.bundle_manifest import BundleManifest
-from urllib.parse import urljoin
 
 
 class BundleRecorder:
     def __init__(self, build, output_dir, artifacts_dir):
         self.output_dir = output_dir
         self.build_id = build.id
-        self.public_url = os.getenv('PUBLIC_ARTIFACT_URL', None)
+        self.public_url = os.getenv("PUBLIC_ARTIFACT_URL", None)
```

Comment:
Nit - Since, default value is optional and is None by default, we could omit it or make it clearer by adding the keyword.
````suggestion
self.public_url = os.getenv("PUBLIC_ARTIFACT_URL", default = None)
````

Reply:
Let's leave this for a separate PR.